tests/br_systables/run.sh

```diff
@@ -36,8 +36,9 @@ rollback_modify() {
 check() {
     run_sql "SELECT count(*) from mysql.foo;" | grep 11
     run_sql "SELECT count(*) from mysql.usertable;" | grep 1000
-    run_sql "SHOW TABLES IN mysql;" | grep -v bar
-    run_sql "SELECT VARIABLE_VALUE FROM mysql.tidb WHERE VARIABLE_NAME = 'tikv_gc_life_time'" | grep "1h"
+    run_sql "SHOW TABLES IN mysql;" | awk '/bar/{exit 1}'
+    # we cannot let user overwrite `mysql.tidb` through br in any time.
+    run_sql "SELECT VARIABLE_VALUE FROM mysql.tidb WHERE VARIABLE_NAME = 'tikv_gc_life_time'" | grep -v "1h"
```

Comment:
```suggestion
    run_sql "SELECT VARIABLE_VALUE FROM mysql.tidb WHERE VARIABLE_NAME = 'tikv_gc_life_time'" | awk '/1h/{exit 1}'
```

Since `run_sql` could make extra lines like `***** row *****` each time, I think it would be better to replace all `grep -v <pattern>` with `awk '/<pattern>/{exit 1}'` here for preventing false negative?